core/che-core-api-core/src/main/java/org/eclipse/che/api/core/jsonrpc/commons/transmission/SendConfiguratorFromOne.java

```diff
@@ -304,10 +304,10 @@ public void sendAndSkipResult() {
             + pValue
             + ", "
             + "result list items class: "
-            + Void.class);
+            + Double.class);
```

Comment:
FYI, I see string concat on Logger debug while we should give object references

Reply:
This problem came after we had merged client and server code bases related to JSON-RPC because on client we had different logging format rules. There are many places where it is necessary to fix logging format, so I created a separate issue not to mix different kinds of changes - https://github.com/eclipse/che/issues/6119